.github/workflows/move-ready-for-review-prs.yml

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: Move PR To Initial Review
+on:
+  issue_comment:
+    types: [created]
+    branch:
+      - main
+jobs:
+  move-pr-to-initial-review:
+    if: github.event.comment.body == '/pr mark ready' && github.event.issue.pull_request
+    runs-on: ubuntu-latest
+    steps:
+      - name: Move To Initial Review
+        uses: actions/github-script@v6
+        with:
+          script: |
+            // Find "Code Reviews" project manually by name matching
+            // This avoids hardcoding the project ID
+            const projects = await github.rest.projects.listForRepo({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+            })
+            const code_reviews = projects.data.find(project => {
+              return project.name === 'Code Reviews';
+            })
+            if (code_reviews === undefined) {
+              return
+            }
+
+            // Find "Initial Review" column manually by name matching
+            // Also find the card of the PR in all columns
+            // This avoids hardcoding the column ID and card ID
+            const columns = await github.rest.projects.listColumns({
+              project_id: code_reviews.id,
+            });
+
+            let work_in_progress = null
+            let pr_card = null
+            // FIXME forEach is needed instead of a for loop, otherwise it doesn't work
+            // FIXME find out why

```

Comment:
Reminder.

Reply:
It works, and the code doesn't look too bad, so this is unnecessary to look into now IMO